tests/manifests/default/kustomization.yaml

```diff
@@ -6,7 +6,6 @@ resources:
   - ./controller_def_1.4.2.yaml
   - ./controller_def_1.4.3.yaml
   - ./controller_def_1.4.4.yaml
-  - ./controller_def_1.5.0-rc1.yaml
```

Comment:
for now, with new RC versions coming, maybe just don't incorporate into kustomization.yaml until there's one we're ready to test with?